src/pages/security/static/_scope_content.ts

```diff
@@ -0,0 +1,67 @@
+export const scope_content = [
+    {
+        title: 'Important businesses',
+        list_icon: 'tick',
+        description: '',
+        check_list: [
+            { content: 'Our payment site: cashier.deriv.com' },
+            { content: 'Our login site: oauth.deriv.com' },
+            { content: 'Our WebSockets API: *.binaryws.com api.deriv.com' },
+            {
+                content: 'Our main trading platform: app.deriv.com<3>*</3>',
+                description: '<0>*</0>This only covers the functionalities handled by Deriv',
+            },
+            { content: 'Our legacy trading platform: smarttrader.deriv.com' },
```

Comment:
🔴  You'll need to include each one of these strings in `localize` in order for it to be extracted out for translation.